src/Apps/Collect/Components/Base/CollectArtworkGrid.tsx

```diff
@@ -55,7 +55,8 @@ class CollectArtworkGrid extends Component<Props, LoadingAreaState> {
         const { state } = filters
         const urlFragment = urlFragmentFromState(state, { page })
 
-        // TODO: Look into using router push w/ query params.
+        // Using window.history.pushState instead of router.push, because
+        //  we just want to add to the history, not navigate to another route.
```

Comment:
#so-absurdly-minor: there's an extra space before the `we`

Reply:
That makes sense – I don't have a strong opinion on starting subsequent comment lines with spaces, personally. We could use two spaces would make it more obvious that this is done with intention – a single space kind of jumps out to me, like it's in the uncanny valley of whitespace.

Reply:
Resolved with Ash's suggestion of adding _more_ space, to escape the uncanny valley.

Reply:
Ha, that extra space is intentional. I prefer my multi-line comments to have that indentation after the first line, so you can clearly identify paragraphs. I think it helps readability.

I'm happy to change this if you think it's a big deal, but it was done with intention.